speech/grpc/src/main/java/com/examples/cloud/speech/StreamingRecognizeClient.java

```diff
@@ -76,6 +81,10 @@ public StreamingRecognizeClient(ManagedChannel channel, String file, int samplin
     this.channel = channel;
 
     speechClient = SpeechGrpc.newStub(channel);
+
+    //Send log4j logs to Console
```

Comment:
I think letting the logger log to the default destination is fine for the sample. If there's a reason that messages should go to stdout, then it should print to stdout rather than logging.

Semantics are important for samples. If you're logging something, then it should be something that shows up in logs. If you're printing something, then it should be communicating with the user. Sending logs to stdout sends sort of mixed messages, IMO.